innovation.game.php

```diff
@@ -7300,6 +7300,22 @@ function stPlayerInvolvedTurn() {
                 self::setGameStateValue('card_id_3', self::executeDraw($player_id, $top_green_card_age, 'hand')['id']);
                 $step_max = 2; // --> 2 interactions: see B
                 break;
+				
+            // id 117, Artifacts age 1: Electrum Stater of Efesos
+            case "117N1":
+                while(true) {
+                    $card = self::executeDraw($player_id, 3, 'revealed'); // "Draw and reveal a 1"
+                    $top_card = self::getTopCardOnBoard($player_id, $card['color']);
+                    if ($top_card == null) 
+					{ // "If you do not have a top card of the drawn card's color"
+                        self::transferCardFromTo($card, $player_id, 'board', false, true); // "meld it"
+                        continue; // "Repeat this effect"
+                    }
+                    self::notifyGeneralInfo(clienttranslate('There was a top card of matching color.'));
```

Comment:
This effect seems straightforward enough that we could do away with this `notifyGeneralInfo` call. These calls are particularly useful when it's hard to trace the logic of what is happening.

Reply:
done